src/main/scala/org/scalacheck/Prop.scala

```diff
@@ -806,16 +907,18 @@ object Prop {
 
   /** Universal quantifier for an explicit generator. Shrinks failed arguments
    *  with the default shrink function for the type */
+  @deprecated("Use 'forAllNoShrink', shrinking will be disabled for Gen, use forAllShrink instead", "1.14.1")
```

Comment:
The deprecation message still needs tweaking.  As I suggested, it should be

    Shrink will be disabled with explicit Gen, use forAllShrink instead

The version number should be `1.15.0`, not `1.14.1`.

Reply:
I'm not sure how strict ScalaCheck is with semver, but I understand you increment the minor version when there is either new functionality or new deprecations.

Reply:
It's not necessary to suggest `forAllNoShrink`, because that's what `forAll` will effectively become.

Reply:
The tone of the deprecation message could be less commanding.  Users could assume they have to migrate to `forAllShrink`.  It's really their choice.

Reply:
@ashawley Thanks, I wasn't paying close enough attention (again). I've just used your full/original suggestion and updated the version number. I've updated the PR. Sorry for the confusion and wasted time.